schema/__tests__/gene.test.js

```diff
@@ -269,6 +269,84 @@ describe("Gene", () => {
     })
   })
 
+  describe("similar", () => {
+    const Gene = schema.__get__("Gene")
+
+    beforeEach(() => {
+      Gene.__ResetDependency__("gravity")
```

Comment:
I think since you're already in the `Gene` file you can refactor the legacy loader to use a proper, cached `genesLoader`. Additionally, you can use the trick in https://github.com/artsy/metaphysics/blob/8acfcc2158faf270e2f4069a8843577781300cad/schema/gene.js#L154 to skip fetching the gene entirely if you're just querying similar genes- up to you.

Reply:
On second thought, that might be a bigger change than needed in this PR (keeping in mind the 'one issue per PR'), so I think this is totally fine as-is for this PR.